defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 
 docker:
+  bridge_ip: 172.17.0.1/16
```

Comment:
Why?

Reply:
I think that is the Docker default? Perhaps we should blank it by default and leave out of the config.

Reply:
Alrighty.. I can default it to none